### ISSUE TITLE
Blood-drunker miner NPC deals no damage to mechas

### ISSUE THREAD

**Issue by Maroonenahmoron** [HippieStation/HippieStationdeprecated2020#9952]
Issue reported from  Round ID: 34672 (Hippie Station)

Reporting client version: 512

I was watching a lad attacking a blood-drunk miner and despite it using the cleaving saw on the mech, no damage or feedback could be seen in the chat.

**Comment by mikolaj007** [comment]
blood drunk miners deal 6-10 damage per hit and 20 per shot.
i think mechas have a damage minimum of 20 or smth.
so either buff miners or nerf mecha

**Comment by Maroonenahmoron** [comment]
Ripley/mechas does/do not ignore damage at a threshold but has a flat chance to deflect attacks. It's understandable that it's immune to legion spawn attacks but not the **melee** attacks of a drunk miner.

Interestingly enough, taking control of the mob and then attacking the mech allows the mech to be damaged. Kinetic shots always affects the mech.

**Comment by Manu9123** [comment]
no nerf faunas #nerflegion

**Comment by xTrainx** [comment]
not true, blood drunk miner recked my shit with a fully equipped durand and i'm pretty sure i wasn't even close
